edi_webservice_oca/models/edi_backend.py

```diff
@@ -33,3 +35,10 @@ def _component_sort_key(self, component_class):
         return (
             1 if getattr(component_class, "_webservice_protocol", False) else 0,
         ) + res
+
+    def _swallable_exceptions(self):
+        exceptions = super()._swallable_exceptions()
+        self.ensure_one()
+        if self.webservice_backend_id.save_response:
```

Comment:
if you swallow it the job state will always be "done"... I'm not sure is desirable :thinking: 
See my comments on https://github.com/OCA/web-api/pull/22

Reply:
Yes job task is done and the EDI exchange is in `output_error_on_send` (as demonstrated by following test bellow `test_component_send_save_results_on_http_error`.

this is because the existing edi code: https://github.com/OCA/edi/blob/14.0/edi_oca/models/edi_backend.py#L297

Do you think the queue job state should be in error as well?